### PR TITLE
Tolu/feat: UI-fixes

### PR DESCRIPTION
**Pull request by blayziken**

### CODE REVIEW

File: lib/constants/custom_color.dart [hunk 1]
```diff
@@ -11,14 +11,14 @@ const onboardHeadingColor = Color(0xFF032E9A);
...
 
-Color lightgreenshede = Color(0xFFF0FAF6);
-Color lightgreenshede1 = Color(0xFFB2D9CC);
-Color greenshede0 = Color(0xFF66A690);
-Color greenshede1 = Color(0xFF93C9B5);
-Color primarygreen = Color(0xFF1E3A34);
-Color grayshade = Color(0xFF93B3AA);
-Color colorAcent = Color(0xFF78C2A7);
-Color cyanColor = Color(0xFF6D7E6E);
+Color lightgreenshede = const Color(0xFFF0FAF6);
```
Comment (yash-garg) [inline lib/constants/custom_color.dart]: Shouldn't these be `const Color lightgreenshede` instead?

**Review by yash-garg — CHANGES_REQUESTED** [review]

File: lib/constants/custom_color.dart (hunk 1 above)
Comment (blayziken) [reply lib/constants/custom_color.dart]: Hmm, you're right. I used the auto-fix linter on vs code.
I'll fix them myself.

Comment (sanjh) [inline lib/constants/custom_color.dart]: remove const

**Review by sanjh — CHANGES_REQUESTED** [review]

File: lib/constants/custom_color.dart (hunk 1 above)
Comment (blayziken) [reply lib/constants/custom_color.dart]: It has been fixed

**Review by yash-garg — APPROVED** [review]

**Pull request merged**